build-release-tools/application/jump_version.py

```diff
@@ -83,24 +108,23 @@ def update_changelog(self, message=None):
         if not debian_exist:
             # Handle repository which contains debianstatic/repository_name folder, 
             # for example: debianstatic/on-http
-            for filename in os.listdir(self._repo_dir):
-                if filename == "debianstatic":
-                    debianstatic_dir = os.path.join(self._repo_dir, "debianstatic")
-                    for debianstatic_filename in os.listdir(debianstatic_dir):
-                        if debianstatic_filename == repo_name:
-                            debianstatic_repo_dir = "debianstatic/{0}".format(repo_name)
-                            common.link_dir(debianstatic_repo_dir, "debian", self._repo_dir)
-                            linked = True
+            linked = self.link_debianstatic()
```

Comment:
May error thrown in ```link_debianstatic``` and handle here is better than ```true/false```?
error can contain more information.